spec/tapioca/dsl/compilers/action_controller_helpers_spec.rb

```diff
@@ -160,13 +160,7 @@ class UserController
                   def helpers; end
 
                   module HelperMethods
-                    include ::ActionController::Base::HelperMethods
-
-                    sig { returns(T.untyped) }
-                    def current_user_name; end
-
-                    sig { params(user_id: ::Integer).void }
```

Comment:
Removing the duplication is nice but I don't think it's worth losing the typing information in the signature.

Reply:
Oh... I see what you mean. I didn't notice in this case the signature for `notify_user` was different in the base class. I'll see how can I avoid this case.